packages/perseus/src/util/graphie.ts

```diff
@@ -1675,25 +1675,57 @@ const setLabelMargins = function (span: HTMLElement, size: Coord): void {
             marginTop: -height / 2 - y * scale,
         });
     } else {
-        const $container = $span.closest(".svg-image");
+        // We are using jQuery to collect information and calculate a scale
+        //     since we don't have a way to pass it to this function.
+        // We need the width of the container in order to calculate the scale to apply to the label.
+        // Unfortunately, the DOM tree leading to the label is not consistent.
+        // Therefore, we need to check different elements and different style properties to get the width of the container.
+        // Some graphie labels exist within an ".svg-image" container, and others do not.
+        // All graphie labels have a ".graphie" container.
+        // When a label has an ".svg-image" container,
+        //     the width information in the ".graphie" element is expressed as a percentage instead of pixels,
+        //     so we can't use it.
+        // Instead, the ".svg-image" container can be used to get the width information.
+        // The following lines determine which container element to use for the width measurement.
+        const $svgImage = $span.closest(".svg-image");
+        const $graphie = $span.closest(".graphie");
+        // If an ".svg-image" container exists in the DOM tree, then use it, otherwise use the ".graphie" container.
+        const $container = $svgImage.length > 0 ? $svgImage : $graphie;
```

Comment:
I'm not certain it's cleaner or clearer, but what do you think about collapsing these three lines to simply:

```typescript
const $container = $span.closest(".svg-image, .graphie")
```

That should select the closest element that matches either selector from what I read.

Reply:
I had that originally, but it doesn't give the needed results. Since there is always a `.graphie` ancestor that is "closer" than a possible `.svg-image`, the query would always return the `.graphie` container. We need the `.svg-container` if it exists, hence the need to split these queries out separately.